package.json

```diff
@@ -100,7 +100,9 @@
     "lodash": "4.11.1",
     "moment": "2.13.0",
     "mqtt": "1.8.0",
+    "node-red-habanero": "^1.0.10",
```

Comment:
i understand doing a specific version for redis, but we'd like to keep the `^` for node-red-habanero, as we will want any deploy to pick up new stuff we do in habanero,  without the need to update concaria